peachjam/helpers.py

```diff
@@ -70,6 +70,8 @@ def to_python(self, value):
 
     def to_url(self, value):
         # invalid values will raise ValueError which will raise NoReverseMatch
+        if isinstance(value, datetime):
```

Comment:
Minor: we're now repeating the same code in two places. A better approach would be to get the value into the expected format (datetime) if it's not there already:

```python
if isinstance(value, str):
    value = datetime.strptime(value, "%Y-%m-%d")
elif not isinstance(value, (datetime, date)):
    raise ValueError(value)
return value.strftime("%Y-%m-%d")
```

Reply:
@longhotsummer agreed, converting the values to datetime and raising errors for all others is a better approach, updated